dags/common/enhance.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+
+
+class Enhancement:
+    def __init__(self):
+        self.creation_date = datetime.datetime.now().isoformat()
```

Comment:
Creation date should not be a property of the enhancer IMO. We should declare this at the start of the __call__ function and pass it to the functions when needed